src/Paket.Core/ProjectFile.fs

```diff
@@ -941,6 +941,9 @@ module ProjectFile =
                 project.ProjectNode.AppendChild analyzersNode |> ignore
             )
 
+    let touch (project:ProjectFile) =
+        if File.Exists(project.FileName) then
```

Comment:
You maybe right about the way how it was ever accessed, but my comment was based on the how the logic is written in the save function and how it changed in the touch. I mean `touch` was intended to be equivalent to `save true`, right? No matter in which context is ever called.